src/leiningen/new/cryogen/themes/blue/html/base.html

```diff
@@ -95,5 +95,7 @@
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
 {% script "js/highlight.pack.js" %}
 <script>hljs.initHighlightingOnLoad();</script>
+{% if post.klipse %} {{post.klipse|safe}} {% endif %}
+{% if page.klipse %} {{page.klipse|safe}} {% endif %}
```

Comment:
I think users can overcome this by adding their own css so let's leave it alone then. Trying to accommodate this use case sounds like it's more work than it's worth.